plugins/inputs/jolokia/README.md

```diff
@@ -1,6 +1,6 @@
 # Jolokia Input Plugin
 
-### Deprecated in version 1.5: Please use the [jolokia2](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/jolokia2) plugin.
+### Deprecated in version 1.5: Please use the [jolokia2][] plugin
```

Comment:
@sjwang90 - this should be `##` as headers should only increment or decrease by 1 level. As this is right after the top level it should only have two.

Reply:
```suggestion
## Deprecated in version 1.5: Please use the [jolokia2][] plugin
```